assemblyline_v4_service/updater/updater.py

```diff
@@ -397,21 +421,20 @@ def do_source_update(self, service: Service) -> None:
                             self.log.info(f'Already visited {uri} in this run. Using cached download path..')
                             output = seen_fetches[uri]
                         else:
-                            # Pull sources from external locations (method depends on the URL)
-                            try:
+                            self.log.info(f"Fetching {source_name} using {fetch_method}")
+                            # Pull sources from external locations
+                            if uri.startswith("file://"):
+                                # Perform an update using a local mount
+                                output = uri.split("file://", 1)[1]
```

Comment:
If we are looking for a local file shouldn't this check for `'file:///'`? A file url still has a host component before the path

Reply:
Updated, thanks!